appimagebuilder/modules/deploy/files/deploy_helper.py

```diff
@@ -111,7 +111,8 @@ def deploy(self, paths: [str]):
             expanded_list = expanded_list.union(glob.glob(path, recursive=True))
 
         for path in expanded_list:
-            self._deploy_path(path)
+            if os.path.isfile(path):
```

Comment:
I guess that it would be better to do this check just before `shutil.copy2` and if it's a directory just call to mkdir. 

https://github.com/AppImageCrafters/appimage-builder/blob/2c668fb6ee4eddbf0d344693894aa2fdbbc4ddb3/appimagebuilder/modules/deploy/files/deploy_helper.py#L122

Reply:
Ok, good point.